app/app.js

```diff
@@ -0,0 +1,36 @@
+const express = require("express");
+const { getAllTopics } = require("./app.controller");
+
+const app = express();
+app.use(express.json());
```

Comment:
you don't need this til you are posting or patching!  Avoid boiler plate code.  Pls delete then add later